dev-requirements.txt

```diff
@@ -1,5 +1,5 @@
 ROCmSoftwarePlatform/rocm-recipes
-RadeonOpenCompute/rocm-cmake@839cf61c355f997283e3baaf245fa9acf2e3baa0 --build
+RadeonOpenCompute/rocm-cmake@695c55dfa4eb3f46a79f3a44620c4cc689b7968c --build
```

Comment:
You should wait until https://github.com/RadeonOpenCompute/rocm-cmake/pull/96 is merged and take that version. I'm just waiting on staging build results.

Reply:
Please use https://github.com/RadeonOpenCompute/rocm-cmake/commit/04f694df2a8dc9d7e35fa4dee4ba5fa407ec04f8 instead. It contains an important bugfix.

Reply:
If at all possible, a version instead of a hash string would be appreciated.